scarpe.gemspec

```diff
@@ -37,6 +37,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "logging", "~>2.3.1"
   spec.add_dependency "webview_ruby", "~>0.1.1"
+  spec.add_dependency "bloops", "~>0.5"
```

Comment:
Caught this. Since we `require` the external dependency in scarpe.rb We'll need it as a runtime dependency. Otherwise folks without this will get

```
`require': cannot load such file -- bloops (LoadError)
	from <internal:/opt/rubies/3.2.0/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /Users/schwad/.gem/ruby/3.2.0/gems/scarpe-0.2.0/lib/scarpe.rb:24:in `<top (required)>'
	from <internal:/opt/rubies/3.2.0/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from <internal:/opt/rubies/3.2.0/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /Users/schwad/.gem/ruby/3.2.0/gems/scarpe-0.2.0/exe/scarpe:94:in `<top (required)>'
	from /Users/schwad/.gem/ruby/3.2.0/bin/scarpe:25:in `load'
	from /Users/schwad/.gem/ruby/3.2.0/bin/scarpe:25:in `<main>'
```

Reply:
They used to have runtime dependencies and development dependencies. But I'm not sure they do any more? We'll have the same problem with logging if you're right, since that's also a runtime dependency.

Reply:
You could try building and running without the "bundle exec" from a locally-installed gem. But I was pretty sure that worked. And even our really basic dependencies like webview just use add_dependency, and have since the beginning.

Reply:
I think there's "add_dependency" and "add_development_dependency" now? Is that wrong?

I secretly pushed 0.2.0 to test it locally when I found this bug running "Scarpe examples/foo.rb". Fails on the require. 

So will need to bump to 0.2.1 and try it out again with this.

Reply:
Clever deploy strategy for bloops fork btw

Reply:
Huh. I think it used to be "add_runtime_dependency" vs "add_development_dependency". I *thought* it was all just "add_dependency" now (a dev-only dependency goes in your Gemfile) but I might be wrong.